recipe/meta.yaml

```diff
@@ -54,12 +54,12 @@ requirements:
     - tbb  >=2021.0
```

Comment:
I think this needs pinning as `tbb 2021.*`. xref: https://github.com/numba/numba/blob/release0.55/buildscripts/condarecipe.local/meta.yaml#L52

Reply:
Does this need to be pinned, or is is just a "pin everything" pin? This: https://github.com/numba/numba/commit/5dfaa6e7c6b26c35efa6aeb0f5afa6df034b30a0 makes me think it might have just been arbitrarily pinned, [a practice I'm not at all fond of](https://iscinumpy.dev/post/bound-version-constraints/). We can always add a pin afterwards in conda - this is not PyPI, you can inject pins without new releases. If there's a failure - are you sure we should pin it here?

Reply:
Okay, as long as there is some reason to believe it will surely break (there are some changes there), then this is fine. If there's a chance it will work, it's better to add the pin later for conda-forge.

Reply:
I think Numba is waiting for packages to be made available to be able to assess what works/does not work in practice, with reasonably high potential for there to be issues. If you'd rather leave it and pin later pending the outcome of the investigation that also seems a reasonable approach, I just wanted to make sure that it wasn't missed as a potential issue. Thanks for your input on this.

Reply:
We could leave off the run_constrained and testing limits, and leave the limit on when building. From what I understand in the linked issue, building on an older TBB should still run with a newer TBB?

Reply:
Yeah we will need to a repodata hot-fix

https://github.com/conda-forge/conda-forge-repodata-patches-feedstock